argilla-server/src/argilla_server/models/database.py

```diff
@@ -236,6 +249,47 @@ def __repr__(self):
             f"inserted_at={str(self.inserted_at)!r}, updated_at={str(self.updated_at)!r})"
         )
 
+    class Select:
```

Comment:
Not sure about this class. Aren't we adding too much complexity only to list some records?